src/main/java/com/checkmarx/flow/config/RepoProperties.java

```diff
@@ -47,6 +54,10 @@ public String getWebhookToken() {
         return webhookToken;
     }
 
+    public String getConfigWebhookToken(ControllerRequest controllerRequest) {
+        return scmConfigOverrider.determineConfigWebhookToken(this, controllerRequest);
```

Comment:
Using the word "effective" in method names would be clearer, e.g. "getEffectiveWebhookToken".